templates/php/src/Permissions.php.twig

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace {{ spec.title | caseUcfirst }};
+
+class Permissions
+{
+    public static function read(string $role): string
+    {
+        return 'read("' $role . '")';
```

Comment:
We are missing the string concat operator in all the PHP methods:

`'read("' $role . '")'`
should be
`'read("' . $role . '")'`

Reply:
done 👍